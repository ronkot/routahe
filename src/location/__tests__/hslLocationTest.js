import {getLocationByQuery} from '../hslLocation'

describe('HSL Location', function() {
  jasmine.DEFAULT_TIMEOUT_INTERVAL = 50000
  it('should fetch location by given criteria', async() => {
    const locations = await getLocationByQuery('Helsingin päärautatieasema')
    expect(locations).toMatchSnapshot()
  })

  it('should return empty object if criteria is empty', async() => {
    const locations = await getLocationByQuery('')
    expect(locations).toEqual(null)
  })
})