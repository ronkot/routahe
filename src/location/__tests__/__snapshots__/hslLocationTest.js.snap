exports[`HSL Location should fetch location by given criteria 1`] = `
Object {
  "label": "Helsingin päärautatieasema, Helsinki",
  "lat": 60.17148,
  "lon": 24.941466,
}
`;
